neat.html

```diff
@@ -162,7 +163,11 @@ <h3>Grid</h3>
 
     <h3>Centering</h3>
 
-    <p>You can throw the <em>center</em> class on almost anything you want to center, including a <em>div</em> tag.
+    <p>You can throw the <em>center</em> class on almost anything you want to center, including a <em>div</em> tag.</p>
+    
+    <h2 id="examples">Examples</h2>
+
+    <p>Example uses of Neat can be found <a href="https://github.com/codazoda/neatcss-examples">here</a>.</p>
```

Comment:
Thanks for the bug fix and the new section. There's a small problem with the link...

Please improve the accessibility of this change by adding additional context instead of the word, "here". For example, you might say something like, "Example uses of Neat can be found in the example repository" and then link the words, "example repository".

Ref: [Why Your Links Should Never Say Click Here](https://uxmovement.com/content/why-your-links-should-never-say-click-here/)